api/konnect/v1alpha1/konnect_gateway_controlplane_types.go

```diff
@@ -61,6 +61,9 @@ type KonnectGatewayControlPlaneSpec struct {
 type KonnectGatewayControlPlaneStatus struct {
 	KonnectEntityStatus `json:",inline"`
 
+	// +kubebuilder:validation:Optional
+	Endpoints *KonnectEndpoints `json:"konnectEndpoints,omitempty"`
```

Comment:
Can we add a comment to this field?

Reply:
See d9e5232a16a3aef593d294c6cec6213df589c785